sonar-jproperties-plugin/src/main/java/org/sonar/plugins/jproperties/issuesaver/crossfile/MissingTranslationsCheckIssueSaver.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * SonarQube Java Properties Plugin
+ * Copyright (C) 2015-2016 David RACODON
+ * david.racodon@gmail.com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.plugins.jproperties.issuesaver.crossfile;
+
+import java.io.File;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.sonar.jproperties.checks.LocaleUtils;
+import org.sonar.jproperties.checks.generic.MissingTranslationsCheck;
+import org.sonar.plugins.jproperties.api.visitors.issue.FileIssue;
+import org.sonar.plugins.jproperties.issuesaver.IssueSaver;
+
+public class MissingTranslationsCheckIssueSaver extends CrossFileCheckIssueSaver {
+
+  public MissingTranslationsCheckIssueSaver(IssueSaver issueSaver) {
+    super(issueSaver);
+  }
+
+  public void saveIssues() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)